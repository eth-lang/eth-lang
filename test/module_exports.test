module Hello (hello world) where
