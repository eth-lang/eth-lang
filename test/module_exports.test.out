test/module_exports.test:0:0 module("Hello") [hello world]

